app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,14 @@
 import * as R from "ramda";
 
+const toCamelCase = str =>
+  str
+    .toLowerCase()
+    .replace(/[^a-zA-Z0-9]+(.)/g, (match, chr) => chr.toUpperCase());
```

Comment:
So if I provide `UX-Designer` as the input then what would be the output from this util function?

My expectation is `uxDesigner`

Reply:
@yedhink yes

Reply:
@yedhink if the input is `UX-Designer`. The output will be `uxDesigner`.